src/ots.cc

```diff
@@ -762,12 +762,18 @@ bool ProcessGeneric(ots::FontFile *header,
   ots::Table *loca = font->GetTable(OTS_TAG_LOCA);
   ots::Table *cff  = font->GetTable(OTS_TAG_CFF);
   ots::Table *cff2 = font->GetTable(OTS_TAG_CFF2);
+  ots::OpenTypeMAXP *maxp = static_cast<ots::OpenTypeMAXP*>(
+    font->GetTypedTable(OTS_TAG_MAXP));
 
   if (glyf && loca) {
     if (font->version != 0x000010000) {
       OTS_WARNING_MSG_HDR("wrong sfntVersion for glyph data");
       font->version = 0x000010000;
     }
+    if (!maxp->version_1) {
+      return OTS_FAILURE_MSG_TAG("wrong maxp version for glyph data",
+                                 OTS_TAG_MAXP);
+    }
```

Comment:
So if there is no `loca`/`glyf`, and there is no `cff` or `cff2`, we accept whatever version we have? Would a bitmap-only color font need a particular version? (or may be since these do not work on Windows any way, it does not matter).

Reply:
It seems unlikely anything except `num_glyphs` would be relevant for such a font, so either version should be adequate. Whether any engine checks the version number is another matter... if so, we might want to set it accordingly, but for now I'd be inclined to just leave it untouched.

Reply:
Makes sense.